charts/keycloakx/README.md

```diff
@@ -482,46 +482,18 @@ extraVolumes: |
 
 ### Changing the Context Path
 
-By default, Keycloak-X is served under context `/`.
-This can be changed to another context path like `/auth` as follows:
+By default, Keycloak-X is served under context `/auth`.
+Trailing slash is removed from path. This can be changed to another context path like `/` as follows:
 
 ```yaml
-contextPath: /auth
-
-extraEnv: |
-  - name: KC_HTTP_RELATIVE_PATH
-    value: {{ .Values.contextPath }}
-
-livenessProbe: |
-  httpGet:
-    path: {{ if ne .Values.contextPath "" }}/{{ .Values.contextPath }}{{ end }}/
-    port: http
-  initialDelaySeconds: 0
-  timeoutSeconds: 5
-
-readinessProbe: |
-  httpGet:
-    path: {{ if ne .Values.contextPath "" }}/{{ .Values.contextPath }}{{ end }}/realms/master
-    port: http
-  initialDelaySeconds: 10
-  timeoutSeconds: 1
-
-startupProbe: |
-  httpGet:
-    path: {{ if ne .Values.contextPath "" }}/{{ .Values.contextPath }}{{ end }}/
-    port: http
-  initialDelaySeconds: 15
-  timeoutSeconds: 1
-  failureThreshold: 60
-  periodSeconds: 5
+http:
+  relativePath: '/'
 ```
 
-The above YAML references introduces the custom value `contextPath` which is possible because `startupScripts`, `livenessProbe`, `readinessProbe`, and `startupProbe` are templated using the `tpl` function.
-Note that it must not start with a slash.
 Alternatively, you may supply it via CLI flag:
 
 ```console
---set-string contextPath=mycontext
+--set-string http.relativePath=/
```

Comment:
This needs to be documented as a breaking change, e.g. in the upgrade guide section in the readme.

This also requires a minor version bump.

Reply:
It is only a breaking change when migrating from the old Keycloak chart, right? The `http.relativePath` is already available in master, but the readme was not adjusted in the KeycloakX chart.